static/src/javascripts/projects/common/modules/commercial/third-party-tags/audience-science-pql.js

```diff
@@ -1,71 +1,72 @@
-        },
-        section,
-        init = function () {
-            section = sectionPlacements[config.page.section] ? config.page.section : 'default';
-        },
-        load = once(function () {
-            if (config.switches.audienceScienceGateway) {
-                var placements = sectionPlacements[section],
-                    query = urlUtils.constructQuery({
-                        placementIdList: placements.join(','),
-                        cb: new Date().getTime()
-                    }),
-                    url = [gatewayUrl, '?', query].join('');
+    var gatewayUrl = '//pq-direct.revsci.net/pql';
+    var storageKey = 'gu.ads.audsci-gateway';
+    var sectionPlacements = {
+        sport:        ['FKSWod', '2xivTZ', 'MTLELH'],
+        football:     ['6FaXJO', 'ORE2W-', 'MTLELH'],
+        lifeandstyle: ['TQV1_5', 'J0tykU', 'kLC9nW', 'MTLELH'],
+        technology:   ['9a9VRE', 'TL3gqK', 'MTLELH'],
+        fashion:      ['TQV1_5', 'J0tykU', 'kLC9nW', 'MTLELH'],
+        news:         ['eMdl6Y', 'mMYVrM', 'MTLELH'],
+        'default':    ['FLh9mM', 'c7Zrhu', 'Y1C40a', 'LtKGsC', 'MTLELH']
+    };
+    var section = sectionPlacements[config.page.section] ? config.page.section : 'default';
+    var audienceSciencePqlUrl = getUrl();
 
-                return require(['js!' + url], function () {
-                    var asiPlacements = window.asiPlacements;
-                    var segments = storage.local.get(storageKey) || {};
-                    // override the global value with our previously stored one
-                    window.asiPlacements = segments[section];
-                    segments[section] = asiPlacements;
-                    storage.local.set(storageKey, segments);
-                });
-            }
-        }),
+    function getUrl() {
+        var placements = sectionPlacements[section];
+        var query = urlUtils.constructQuery({
+            placementIdList: placements.join(','),
+            cb: new Date().getTime()
+        });
+        return gatewayUrl + '?' + query;
+    }
 
-        getSegments = function () {
-            var segments = {},
-                storedSegments = storage.local.get(storageKey);
-            if (config.switches.audienceScienceGateway && storedSegments) {
-                segments = chain(pairs(storedSegments[section])).and(filter, function (placement) {
-                        //keyword `default` written in dot notation throws an exception IE8
-                        return placement[1]['default']; //eslint-disable-line
-                    }).and(map, function (placement) {
-                        return ['pq_' + placement[0], 'T'];
-                    }).and(zipObject).valueOf();
-                // set up the global asiPlacements var in case dfp returns before asg
-                window.asiPlacements = storedSegments[section];
-            }
-            return segments;
-        };
+    function onLoad() {
```

Comment:
TBH I don't know, seems we swap two values: one we receive from the 3rd party, one we store in local storage.